samples/snippets/iam_test.py

```diff
@@ -119,7 +119,7 @@ def test_check_subscription_permissions(subscription, capsys):
 
 
 def test_detach_subscription(subscription, capsys):
-    iam.check_subscription_permissions(PROJECT_ID, SUBSCRIPTION_ID)
+    iam.detach_subscription(PROJECT_ID, SUBSCRIPTION_ID)
```

Comment:
This method isn't available IAM. We should write this test like this:
```
def test_check_detach_subscription_permission(topic, capsys):
    iam.check_topic_permissions(PROJECT_ID, TOPIC_ID)	

    out, _ = capsys.readouterr()

    assert topic in out
    assert "pubsub.topics.detachSubscription" in out
```

Reply:
iam refers to `/samples/snippets/iam.py`. This is a unit test for the sample only.

Reply:
Oh! Sorry, I thought it was an imported IAM library when I saw the imports in this file.